src/proxy.ts

```diff
@@ -198,25 +198,21 @@ const server = http.createServer(function (req, res) {
       target,
       secure: false,
     });
-
-    proxyApp.on("error", function (err) {
-      console.log("app>>", req.method, target + req.url);
-      res.writeHead(500, {
-        "Content-Type": "text/plain",
-      });
-
-      res.end(err.toString());
-    });
   }
 });
 
+proxyApp.on("error", function (err) {
```

Comment:
Moving this here so the handler is created once and print error to stderr.

This handler was called because the WebSocket target was misconfigured (see below).